### PR TITLE
Added screenshot for Github Markdown

### PR DESCRIPTION
**Pull request by tacensi**
#6 Added screenshots showing Github markdown editor along with the preview. Changed a little bit of the text to describe the image.

### CODE REVIEW

File: _articles/get-your-project-online/the-github-interface-and-markdown.md [hunk 1]
```diff
@@ -37,7 +37,11 @@ Here are some examples of how you can use Markdown.
...
 
-You can find [more documentation on using Markdown on Daring Fireball](http://daringfireball.net/projects/markdown/syntax), [and GitHub has its own markdown](http://daringfireball.net/projects/markdown/syntax) [documented here](https://guides.github.com/features/mastering-markdown/). There are also tools like [MacDown](http://macdown.uranusjr.com/) that show live previews to make writing Markdown easier.
+![Github markdown interface with preview]({{ site.baseurl }}/img/github-markdown.png)
+
+You can preview the message you write before sending using the preview tab in the markdown editor.
+
+Find [more documentation on using Markdown on Daring Fireball](http://daringfireball.net/projects/markdown/syntax), [and GitHub has its own markdown](http://daringfireball.net/projects/markdown/syntax) [documented here](https://guides.github.com/features/mastering-markdown/). There are also tools like [MacDown](http://macdown.uranusjr.com/) that show live previews to make writing Markdown easier.
```
Comment (betatim) [inline _articles/get-your-project-online/the-github-interface-and-markdown.md]: I had to read this a few times before I figured out why we were linking to daringfireball twice. Especially as the second link is "github have their own markdown". I'd propose removing that second link so people wanting to know about github flavoured markdown can only go to https://guides.github.com/features/mastering-markdown/. What do you think?

Comment (abbycabs) [reply _articles/get-your-project-online/the-github-interface-and-markdown.md]: +1 @tacensi, can you update the 2nd link. This looks great though!

**Comment by tacensi** [comment]
Sorry about that, didn't notice the duplicated link. It is fixed.

**Comment by abbycabs** [comment]
Noice! Merging :) Thanks @tacensi 🎉

**Pull request merged**